libs/designsystem/src/lib/components/page/page.component.ts

```diff
@@ -80,6 +82,27 @@ export class PageContentDirective {
   }
 }
 
+@Component({
+  selector: 'kirby-page-progress',
+  template: `
+    <ng-content></ng-content>
+  `,
+  styles: [':host {display: flex}'],
+})
+export class PageProgressComponent implements OnInit {
+  // TODO: Find alternative implementation, which aligns with future page configuration / consumption
+  // this implementation was chosen over expanding moveChild in component wrapper with another scenario
+  @HostBinding('attr.slot') slotAttribute = 'start';
```

Comment:
Consider naming:
```suggestion
  @HostBinding('attr.slot') slotLocation = 'start';
```
OR
```suggestion
  @HostBinding('attr.slot') slotPlacement = 'start';
```
OR
```suggestion
  @HostBinding('attr.slot') toolbarSlot = 'start';
```
OR SIMPLY
```suggestion
  @HostBinding('attr.slot') slot = 'start';
```